### PR TITLE
docs: add OIDC guides for Okta and Keycloak

### PR DESCRIPTION
**Pull request by unknwon**
Close BYT-2336

### CODE REVIEW

File: content/docs/administration/sso/oidc.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ OpenID Connect (OIDC) is a simple identity layer on top of the OAuth 2.0 protoco
 
 <hint-block type="info">
 
-Please make sure the [`--external-url`](/docs/get-started/install/external-url) is configured correctly for the Bytebase instance.
+1. Please make sure the [`--external-url`](/docs/get-started/install/external-url) is configured correctly for the Bytebase instance.
+   
+   If your start Bytebase with `--external-url https://bytebase.example.com`, then your application redirect URL should be `https://bytebase.example.com/oidc/callback`.
 
-If your start Bytebase with `--external-url https://bytebase.example.com`, then your application callback or redirect URL should be `https://bytebase.example.com/oidc/callback`.
+2. If you're unsure about the **Issuer** of your IdP, you can always use the [OpenID Connect Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html) endpoint to find the correct value, e.g. `https://dev-10086.okta.com/.well-known/openid-configuration`.
```
Comment (tianzhou) [inline content/docs/administration/sso/oidc.md:15]: acme.okta.com ?

provided that acme represents the tenant?

File: content/docs/administration/sso/oidc.md [hunk 2]
```diff
@@ -69,3 +71,35 @@ In some GitLab self-hosted setups, the **Issuer** is `http://gitlab.example.com`
...
 </hint-block>
 
+### Okta
+
+1. Follow the Okta [create OIDC app integrations documentation](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm) to create a new OIDC app integration with "Web Application" as the **Application type**.
+1. Configure the **Sign-in redirect URIs** to be `{EXTERNAL_URL}/oidc/callback`.
+1. In Bytebase, go to **Settings > SSO** to create a new OIDC provider (all values are examples):
+   - **Name**: `Okta`
+   - **Resource ID**: `okta`
+   - **Domain**: `bytebase.okta.com`
```
Comment (tianzhou) [inline content/docs/administration/sso/oidc.md:81]: I may choose acme, same for the keycloak

Comment (tianzhou) [inline content/docs/administration/sso/oidc.md:81]: I may choose acme.okta.com, same for the keycloak

File: content/docs/administration/sso/oidc.md (hunk 1 above)
Comment (unknwon) [inline content/docs/administration/sso/oidc.md:15]:
```suggestion
2. If you're unsure about the **Issuer** of your IdP, you can always use the [OpenID Connect Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html) endpoint to find the correct value, e.g. `https://acme.okta.com/.well-known/openid-configuration`.
```

File: content/docs/administration/sso/oidc.md (hunk 2 above)
Comment (unknwon) [reply content/docs/administration/sso/oidc.md]: Updated!

File: content/docs/administration/sso/oidc.md [hunk 3]
```diff
@@ -65,7 +67,39 @@ User information field mapping:
...
+1. Follow the Keycloak [create OIDC provider documentation](https://www.keycloak.org/docs/latest/server_admin/#assembly-managing-clients_server_administration_guide) to create a new "OpenID Connect" client.
+   1. Configure the **Client ID** to be `bytebase`.
+   1. Configure the **Valid redirect URIs** to be `{EXTERNAL_URL}/oidc/callback`.
+   1. Turn on the **Capability config > Client authentication** for the **Credentials** tab to be available (which will generate and display the client secret).
+      1. In some older versions, configure **Access Type** to "confidential" instead.
+1. In Bytebase, go to **Settings > SSO** to create a new OIDC provider (all values are examples):
+   - **Name**: `Keycloak`
+   - **Resource ID**: `keycloak`
+   - **Domain**: `keycloak.acme.com`
+   - **Issuer**: `https://bytebase.okta.com/auth/realms/master`
```
Comment (unknwon) [inline content/docs/administration/sso/oidc.md:100]:
```suggestion
   - **Issuer**: `https://acme.okta.com/auth/realms/master`
```

**Comment by unknwon** [comment]
@tianzhou Updated, PTAL!

**Pull request merged**